service/src/test/java/uk/gov/hmcts/reform/fpl/model/AbstractValidationTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validator;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = LocalValidatorFactoryBean.class)
+abstract class AbstractValidationTest {
```

Comment:
There is a class called `ValidatorTest`, would this do something similar to what this class achieves?

Reply:
cool, refactored